rhel/SPECS/dbaas-tools.spec

```diff
@@ -0,0 +1,64 @@
+%define debug_package %{nil}
+
+%global project_aws         kubernetes-sigs
+%global repo_aws            aws-iam-authenticator
```

Comment:
And what it would be once we add kubectl?

Reply:
Few another variables, better not to hardcode

@EvgeniyPatlan thoughts?

Reply:
Why not hard-code a fixed thing? :) Do we place to change it? https://github.com/percona/pmm-server-packaging/pull/160#discussion_r481018902

Reply:
Ok, I'll hardcode it and leave only commit & shortcommit

Reply:
The general practice is to use named variables for things that are:
* variables (i.e. they change their value from time to time);
* are used in more that one place, or in some tricky place where they may not be noticeable (like in the middle of the string with interpolation).

`repo_aws` value will not really change